pymc3/model.py

```diff
@@ -944,15 +944,15 @@ def cont_vars(self):
         return list(typefilter(self.value_vars, continuous_types))
 
     def set_initval(self, rv_var, initval):
-        if initval is not None:
-            initval = rv_var.type.filter(initval)
+        from pymc3.distributions.distribution import default_initval
 
-        test_value = getattr(rv_var.tag, "test_value", None)
+        if initval is None:
+            initval = default_initval(rv_var)
```

Comment:
we can add the old filter here when `initval is not `None` if we want to explicitly disallow symbolic expression from the user